pkg/controller/reconciler/annotations.go

```diff
@@ -44,14 +44,23 @@ type deploymentAnnotations struct {
 	// This will be nil if Radius is not enabled for the Deployment.
 	Configuration *deploymentConfiguration
 
-	//ConfigurationHash is the hash of the user-provided configuration.
+	// ConfigurationHash is the hash of the user-provided configuration.
 	// This will be used to diff the configuration and determine if the Deployment needs to be updated.
 	ConfigurationHash string
 
 	// Status is the status of the Deployment (Radius related status).
 	Status *deploymentStatus
```

Comment:
If Status is not provided via annotations even though Radius is disabled, we have no way of knowing if that Deployment had Radius enabled before.

Reply:
And the only way to know that Radius was previously enabled on the Deployment is to check the Status object I guess, right? @rynowak